src/module.c

```diff
@@ -5824,7 +5824,7 @@ RedisModuleCallReply *RM_Call(RedisModuleCtx *ctx, const char *cmdname, const ch
     /* Lookup command now, after filters had a chance to make modifications
      * if necessary.
      */
-    cmd = c->cmd = c->lastcmd = c->realcmd = lookupCommand(c->argv,c->argc);
+    c->cmd = c->lastcmd = c->realcmd = lookupCommand(c->argv,c->argc);
```

Comment:
I am not sure if we reset `c->cmd` when we release the client (return it to the pool). Maybe its safer that we do?

Reply:
done